ReactSkia/sdk/NotificationCenter.h

```diff
@@ -62,9 +64,11 @@ class NotificationCenter {
 
         NotificationCenter(const NotificationCenter&) = delete;  
         const NotificationCenter& operator = (const NotificationCenter&) = delete;
+        folly::ScopedEventBaseThread eventBaseThread_;
```

Comment:
can we give a name to this thread ? since we have many folly threads in RNS code , it will help in identifying the threads by their name